integration-tests/validation/src/main/java/com/android/designcompose/testapp/validation/examples/AllExamples.kt

```diff
@@ -109,3 +76,40 @@ val EXAMPLES: ArrayList<Triple<String, @Composable () -> Unit, String?>> =
         // GH-636: Test takes too long to execute.
         // Triple("Very large File", { VeryLargeFile() }, VeryLargeFileDoc.javaClass.name)
     )
+
+val SQUOOSH_ONLY_EXAMPLES: ArrayList<Triple<String, @Composable () -> Unit, String?>> =
+    arrayListOf(
+        Triple("SA", { SmartAnimateTest() }, SmartAnimateTestDoc.javaClass.name),
+        Triple("SA Variant", { VariantAnimationTest() }, VariantAnimationTestDoc.javaClass.name),
+    )
+
+val DEFAULT_RENDERER_ONLY_EXAMPLES: ArrayList<Triple<String, @Composable () -> Unit, String?>> =
+    arrayListOf(
+        // Squoosh doesn't work with the doc id override.
+        Triple("HelloBye", { HelloBye() }, HelloByeDoc.javaClass.name),
+
+        // Lazy Grid doesn't actually use a doc
+        Triple("Lazy Grid", { LazyGridItemSpans() }, null),
+        Triple("Grid Layout", { GridLayoutTest() }, GridLayoutTestDoc.javaClass.name),
+        Triple("Grid Widget", { GridWidgetTest() }, GridWidgetTestDoc.javaClass.name),
+        Triple("List Widget", { ListWidgetTest() }, ListWidgetTestDoc.javaClass.name),
+        Triple("Item Spacing", { ItemSpacingTest() }, ItemSpacingTestDoc.javaClass.name),
+        Triple("Layout Tests", { LayoutTests() }, LayoutTestsDoc.javaClass.name),
+        Triple(
+            "Layout Replacement",
+            { LayoutReplacementTest() },
+            LayoutReplacementTestDoc.javaClass.name
+        ),
+        Triple("OpenLink", { OpenLinkTest() }, OpenLinkTestDoc.javaClass.name),
+        Triple(
+            "Recurse Customization",
+            { RecursiveCustomizations() },
+            RecursiveCustomizationsDoc.javaClass.name
+        ),
+        Triple("Shared Customization", { ModuleExample() }, ModuleExampleDoc.javaClass.name),
+        Triple(
+            "Variant Interactions",
+            { VariantInteractionsTest() },
+            VariantInteractionsTestDoc.javaClass.name
+        ),
+    )
```

Comment:
Did you manually test each validation test in squoosh and non-squoosh mode to see which ones squoosh had problems with?

Reply:
I did test each validation test but I didn't looked deep into the differences.
Squoosh only tests have the two tests for animations and default collection has those with list/grid layout which squoosh doesn't support yet.
The set we do in both squoosh and non-squoosh, there are still some differences which I believe there are bugs with squoosh there: for example squoosh doesn't cut off the texts into two lines with the Text Elide test.

Reply:
Got it. Would be nice to get a comparison of squoosh against non-squoosh sometime, and I know @iamralpht started a doc with a table describing what is different or broken in squoosh. Not for this PR though.